includes/class-multisite-directory-taxonomy.php

```diff
@@ -133,4 +135,32 @@ public function saveTermGeo ($term_id) {
         }
     }
 
+    /**
+     * Adds a dropdown box to filter directory entries by category in the admin listing
+     *
+     * @link https://developer.wordpress.org/reference/hooks/restrict_manage_posts/
+     *
+     * @param string $post_type
+     */
+    public static function add_taxonomy_admin_filter ($post_type) {
+
+        if (Multisite_Directory_Entry::name === $post_type) {
+            $tax_obj = get_taxonomy(self::name);
+            $tax_name = $tax_obj->labels->name;
+            $terms = get_terms(self::name);
+            if (count($terms) > 0) {
+                echo '<select name="' . esc_attr(self::name) . '" id="' . esc_attr(self::name) . '" class="postform">';
```

Comment:
Ah, yeah, I thought so. It looked like cruft. Sure, remove it in another commit, but since [you'll be squashing anyway](https://github.com/meitar/multisite-directory/pull/23#issuecomment-269633517), you don't need to push that one commit.